appveyor.yml

```diff
@@ -22,11 +22,12 @@ install:
   - gem update --system
```

Comment:
Don't you need the travis style ugly dance here to grab the correct version.  Admittedly, I don't fault you for skipping this if you hate CMD.